server/model/block.go

```diff
@@ -62,6 +62,10 @@ type Block struct {
 	// The workspace id that the block belongs to
 	// required: true
 	WorkspaceID string `json:"workspaceId"`
+
+	// Indicates if the card is limited
+	// required: false
+	Limited bool `json:"limited,omitempty"`
```

Comment:
I think it was `IsLimited`, it is from the client side, I don't mind to change that in my side anyway, but we should be consistent.

Reply:
Let's leave `limited` then for consistency with other branches as well